web/constants/dashboard.ts

```diff
@@ -141,6 +141,10 @@ export const DURATION_FILTER_OPTIONS: {
     key: "this_year",
     label: "Due this year",
   },
+  {
+    key: "custom",
```

Comment:
since the keys are used in conditionals they can be converted into enums and re used. The Logic behind this is if i have to change value here, i'll have to change it everywhere, better for maintainability

Reply:
I have defined types for everything, is it really necessary to use enum?

Reply:
Since it is being used to compare and check if it is custom, it is better to use an enum or constant for better maintainability. tomorrow if i wanted to change the "custom" to "userDefined", I'll have to change every where it is used. If it is an enum i'll only have to change in one place

Reply:
Got it. Thanks.